ttnn/cpp/ttnn/operations/conv/conv2d/prepare_conv2d_weights.cpp

```diff
@@ -71,7 +71,7 @@ ttnn::Tensor conv_bias_layout_convert(
     validate_bias_tensor(bias_tensor_);
     if (!is_non_tile_mul_width) {
         auto bias_shape = bias_tensor_.get_shape();
-        TT_FATAL(bias_shape[3] == out_channels && bias_shape[0] == 1 && bias_shape[1] == 1 && bias_shape[2] == 1, "bias shape is not correct");
+        TT_FATAL(bias_shape[0] == 1 && bias_shape[1] == 1 && bias_shape[2] == 1, "bias shape is not correct");
```

Comment:
What happend to this part of the check?
bias_shape[3] == out_channels

Reply:
out_channels ia actually out_channels_padded, and they may not be equal.

Reply:
can we extend the check to check for that as well (has to be padded or unpadded value), instead of just removing it?

Reply:
I can add a check before this function is called.

Reply:
Done.